src/services/game-logic/GameEngine.ts

```diff
@@ -1,13 +1,16 @@
-import { PieceNames, Constants } from '../../enums';
+import { PieceNames, Constants, SoundTypes } from '../../enums';
 import { Square } from '../../models/Square';
 import { Board } from './Board';
 import { Piece } from './pieces/Piece';
+import { SoundController } from '../../controllers/SoundController';
```

Comment:
Wierzę że to działa ale to podejście psuje nam strukturę projektu. GameEngine ma zajmować się tylko i wyłącznie logiką gry. Ta klasa którą stworzyłaś powinna być wykorzystana w ramach GameControllera

Reply:
ok! Dzięki! Tutaj również miałam wątpliwość na jakim poziomie umieścić wywołanie :)